### ISSUE TITLE
I can not move first list's items to second list when i changed scecond list position

### ISSUE THREAD

**Issue by wakeupSu** [atlassian/react-beautiful-dnd#2070]
### Expected behavior
move succeed
### Actual behavior
move failture
### Steps to reproduce
move first list's items to second list when i changed scecond list position
### Suggested solution?
 no idea

### What version of `React` are you using?
16.8.5

### What version of `react-beautiful-dnd` are you running?
13.0.0

### What browser are you using?
chrome

### Demo
https://codesandbox.io/s/vertical-list-with-multiple-drop-targets-forked-kk1qm?file=/index.js:3476-3485

**Comment by clentfort** [comment]
TL;DR: Use `onBeforeCapture` instead of `onDragStart`

As outlined in the [responder's guide](https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/responders.md) you can not change the dimensions of draggables and droppables once the drag has started. You can only do this in `onBeforeCapture` and in/after `onDragEnd`.

**Comment by wakeupSu** [comment]
> TL;DR: Use `onBeforeCapture` instead of `onDragStart`
> 
> As outlined in the [responder's guide](https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/responders.md) you can not change the dimensions of draggables and droppables once the drag has started. You can only do this in `onBeforeCapture` and in/after `onDragEnd`.

Thank you for your reply. but i have to change the position of the list while dragging. If you have any suggestions, please tell me. Thanks again.

**Comment by clentfort** [comment]
The dimensions include the position and as far as I know there is no way to do it in onDragStart.

**Comment by wakeupSu** [comment]
> The dimensions include the position and as far as I know there is no way to do it in onDragStart.

ok, thanks, not lucky for me...

**Comment by danieldelcore** [comment]
Hey @wakeupSu are you still experiencing this issue?
